### PR TITLE
Improving events page mobile

### PR DESCRIPTION
**Pull request by tiago-lp**
**Feature/Bug description:** Redesigning the events page on mobile devices to show different the events without pictures and deleted events.

**Solution:** 
selection_042

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/event/events_mobile.html [hunk 1]
```diff
@@ -29,7 +29,19 @@
             ng-click="eventCtrl.goToEvent(event)">
                 <md-card>
```
Comment (JuliePessoa) [inline frontend/event/events_mobile.html]: create an "event-card" component

File: frontend/event/events_mobile.html [hunk 2]
```diff
@@ -29,7 +29,19 @@
...
                   <div class="event-preview event-preview-border" md-colors="{color: eventCtrl.getProfileColor(event)}">
-                    <img class="md-card-image event-preview-image" ng-src="{{event.photo_url}}">
+                    <img ng-if="event.state !== 'deleted'" ng-class="event.photo_url ? 'event-preview-image' : 'event-preview-without-image'"
+                    md-colors="{background: eventCtrl.getProfileColor(event)+'-500'}" class="md-card-image" ng-src="{{event.photo_url}}">
+                    <div ng-if="event.state === 'deleted'" class="event-preview-image md-card-image"
```
Comment (JuliePessoa) [inline frontend/event/events_mobile.html]: create an canceled event header component

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/event/events_mobile.html [hunk 3]
```diff
@@ -29,7 +29,19 @@
...
+                    md-colors="{background: eventCtrl.getProfileColor(event)+'-700'}">
+                       <span layout="row">
+                        <md-icon class="event-canceled-icon">highlight_off</md-icon>
```
Comment (brobsc) [inline frontend/event/events_mobile.html]: The cancelled icon looks out of place and overlaps with text on chrome mobile emulator

**Review by brobsc — CHANGES_REQUESTED** [review]

File: frontend/event/events_mobile.html [hunk 4]
```diff
@@ -50,13 +62,13 @@
       </div>
     </md-content>
   </div>
+  <md-fab-speed-dial ng-if="!eventCtrl.institutionKey"
```
Comment (Luiz-FS) [inline frontend/event/events_mobile.html]: Why move?

**Review by Luiz-FS — CHANGES_REQUESTED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Review by brobsc — APPROVED** [review]

**Review by Luiz-FS — APPROVED** [review]

**Pull request merged**